src/fides/api/ops/task/graph_task.py

```diff
@@ -124,11 +140,13 @@ def result(*args: Any, **kwargs: Any) -> Any:
                     )
                     sleep(func_delay)
                     raised_ex = ex
-
             self.log_end(action_type, raised_ex)
             self.resources.request.cache_failed_checkpoint_details(
                 step=action_type, collection=self.traversal_node.address
             )
+            add_errored_system_status_for_consent_reporting(
+                self.resources.request, self.connector.configuration
+            )
```

Comment:
This marks privacy preferences that we intended to propagate to a third party with an errored system status for consent reporting for saas connectors.